app/src/main/java/ingsw/group1/findmyphone/contacts/SMSContactConverter.java

```diff
@@ -0,0 +1,57 @@
+package ingsw.group1.findmyphone.contacts;
+
+import androidx.room.TypeConverter;
+
+import ingsw.group1.msglibrary.SMSPeer;
+
+/**
+ * Class of static methods necessary to convert {@link SMSPeer} to {@link SMSContact} and vice versa.
+ * It doesn't check the validity of a contact or peer.
+ *
+ * @author Giorgia Bortoletti
+ */
+class SMSContactConverter {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'SMSContactConverter' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=4929954)